docs/topics/compatibility-guides/compatibility-guide-21.md

```diff
@@ -0,0 +1,479 @@
+[//]: # (title: Compatibility guide for Kotlin 2.1)
```

Comment:
A related question: would it be possible to link the compatibility guide from the What's new page? (as we did in 1.9 and before)

Reply:
Hi Ilya - I asked the team - we'll link to it! 👍